packages/vital-carousel/src/components/Carousel/utils/carousel.ts

```diff
@@ -0,0 +1,261 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
```

Comment:
Minimal script that can be added via withoutHydration after everything loads:
Working copy is here:  https://codepen.io/Heidi-Solutions-by-Heidi-LLC/pen/jOQzWoq